host/bcm283x/gpio.go

```diff
@@ -75,6 +75,10 @@ var (
 	GPIO46 *Pin //
 	// Pins 47~53 are not exposed because using them would lead to immediate SD
 	// Card corruption.
+
+	// Whether uses the old pull resistor setup method before bcm2711.
+	// If true, older method is used, and Pull() won't give useful information.
+	UseLegacyPull = true
```

Comment:
I'd prefer this to not be exported, since it's not meant to be used by the user. So I recomment to make it a member of driverGPIO.